tests/local_vespa/test_app/services.xml

```diff
@@ -0,0 +1,83 @@
+          - Reference: https://docs.vespa.ai/en/reference/services-container.html
+    -->
+    <container id="default" version="1.0">
+        <!--
+            <document-api> tells the container that it should accept documents for indexing. Through the
+            Document REST API you can PUT new documents, UPDATE existing documents, and DELETE documents
+            already in the cluster.
+
+            Documents sent to the Document REST API will be passed through document processors on the way
+            to the content cluster.
+
+            See:
+             - Reference: https://docs.vespa.ai/en/reference/services-container.html#document-api
+             - Operations: https://docs.vespa.ai/en/document-v1-api-guide.html
+        -->
+        <document-api/>
+
+        <!--
+            <search> tells the container to answers queries and serve results for those queries.
+            Inside the <search /> cluster you can configure chains of "searchers" -
+            Java components processing the query and/or result.
+
+            See:
+             - Reference: https://docs.vespa.ai/en/query-api.html
+             - Searchers: https://docs.vespa.ai/en/searcher-development.html
+        -->
+        <search/>
+
+        <!--
+            <nodes> specifies the nodes that should run this cluster.
+        -->
+        <nodes deploy:environment="dev" count="1">
+            <resources vcpu="2" memory="8Gb" disk="50Gb"/>
+        </nodes>
+        <nodes deploy:environment="perf" count="[1, 4]">
+            <resources vcpu="[2, 4]" memory="[8Gb, 16Gb]" disk="[50Gb, 500Gb]"/>
+        </nodes>
+        <nodes deploy:environment="prod" count="[1, 4]">
+            <resources vcpu="[2, 4]" memory="[8Gb, 16Gb]" disk="[50Gb, 500Gb]"/>
+        </nodes>
+    </container>
+
+    <!--
+        <content/> content clusters store application data, maintain indexes and executes the
+        distributed parts of a query.
+
+        See:
+          - Reference: https://docs.vespa.ai/en/reference/services-content.html
+    -->
+    <content id="family-document-passage" version="1.0">
+        <tuning>
+            <resource-limits>
+                <disk>0.95</disk>
+            </resource-limits>
+        </tuning>
+        <min-redundancy>2</min-redundancy>
+        <documents>
+            <document type="search_weights" mode="index" global="true" />
+            <document type="family_document" mode="index" global="true" />
+            <document type="document_passage" mode="index" />
+        </documents>
+        <nodes deploy:environment="dev" count="1">
+            <resources vcpu="2" memory="16Gb" disk="300Gb"/>
+        </nodes>
+        <nodes deploy:environment="perf" count="[2, 4]">
+            <resources vcpu="[2, 8]" memory="[16Gb, 64Gb]" disk="[300Gb, 1Tb]"/>
+        </nodes>
+        <nodes deploy:environment="prod" count="[2, 4]">
+            <resources vcpu="[2, 8]" memory="[16Gb, 64Gb]" disk="[300Gb, 1Tb]"/>
+        </nodes>
```

Comment:
Same comment here. These are also duplicated from above - do they need to be?

Reply:
Yes, I believe so. My understanding is that the first is for management and configuration of the cluster itself, whereas the second is for queries and data